src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128Helpers.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics.Arm;
+using System.Runtime.Intrinsics.X86;
+
+namespace System.Runtime.Intrinsics
+{
+    // Contains internal helper methods
+    internal static class Vector128Helpers
```

Comment:
Why not just reuse the existing VectorMath for this (renaming it to VectorHelpers if it helps)? Types have overhead too.

Reply:
Done